symbolic-symcache/src/error.rs

```diff
@@ -1,3 +1,4 @@
+use std::error::Error;
```

Comment:
we import two `Error` types now. Can we name `thiserror::Error` directly in the derive?

Reply:
Since one is a derive and one a trait, this is fine. Compare this to `serde`. Since the derive applies to the very same error type, I'd leave it like this.